front/lib/types/workos.ts

```diff
@@ -0,0 +1,8 @@
+export enum WorkOSPortalIntent {
+  SSO = "sso",
+  DSync = "dsync",
+  DomainVerification = "domain_verification",
+  AuditLogs = "audit_logs",
+  LogStreams = "log_streams",
+  CertificateRenewal = "certificate_renewal",
+}
```

Comment:
note: we need our own type because we can not import WorkOS's stuff in the client